src/pages/home/features/Features.m.css

```diff
@@ -0,0 +1,80 @@
+@import '../../../variables.css';
+
+.root {
+}
+
+.featureOne {
+	background: linear-gradient(110deg, var(--color-red), var(--color-purple));
+}
+
+.featureTwo {
+	background: var(--darker-background-color);
+}
+
+.featureThree {
+	background: linear-gradient(110deg, var(--color-purple), var(--color-blue));
+}
+
+.featureRow {
+	flex-direction: row;
+	display: flex;
+}
+
+.featureCardRight {
+	max-width: 750px;
+	height: 550px;
+	margin: 100px auto 100px 0px;
+}
+
+.featureCardLeft {
+	max-width: 750px;
+	height: 550px;
+	margin: 100px 0 100px auto;
+}
+
+.featurePoints {
+	font-size: 22px;
+	display: flex;
+	flex-direction: column;
+	align-items: start;
+	list-style-type: none;
+	margin: 0;
+	padding: calc(var(--grid-size) * 1.5) var(--card-padding) var(--card-padding);
+}
+
+.featurePoints > li {
```

Comment:
BTR does not handle element selectors well when compiling the correct CSS. We should always use class selectors.

Reply:
Yeah, sorry this was me being lazy and not correcting it after I check that it worked!